css/index.css

```diff
@@ -237,6 +240,7 @@ nav a {
 	background-color: #eee;
 }
 
+/*For mobile screen*/       
 @media (max-width: 768px){
     .contentBox{
         margin-bottom: 1rem;
```

Comment:
comment your code is a good practice ! I really liked that you comment those lines of code